pages/text-diff.tsx

```diff
@@ -33,12 +104,19 @@ export default function TextDiffPage() {
   function handleChange2(event: React.ChangeEvent<HTMLInputElement>) {
     setInput2(event.target.value);
   }
+  function handleSelectChange(event: SelectChangeEvent<any>) {
```

Comment:
Don't use `any`, please run `npm run cq` to check for compiler/linter errors and address anything that comes up.